### DISCUSSION TITLE
Does it work with the latest tensorflow version 2.6?

### DISCUSSION THREAD

**Discussion by RomanFoell** [Rishit-dagli/TF-Watcher#99] [General]
Hello,

when I want to install the package via pip tensorflow 2.5 gets downloaded.
Does this mean, there is no support for tensorflow 2.6?

Best, Roman

**Comment by Rishit-dagli** [comment]
Hi @RomanFoell ,
TL: DR Yes, it does.
As of now, the dependency requirements in the **pip release** are ~=2.5 but I will fix that in the upcoming patch (#100 ). Until then you could most certainly install TensorFlow 2.6 and it should work well.

**Comment by Rishit-dagli** [comment]
@RomanFoell Recently released patch [v0.1.2](https://pypi.org/project/tf-watcher/0.1.2/) fixes this, please use the latest version.
